bodyfetcher.py

```diff
@@ -208,12 +239,39 @@ def make_api_call_for_site(self, site):
                 owner_rep = 0
             q_id = str(post["question_id"])
 
-            is_spam, reason, why = check_if_spam(title, body, owner_name, owner_link, site, q_id, False, False, owner_rep, post_score)
+            is_spam, reason, why = check_if_spam(
+                title,
+                body,
+                owner_name,
+                owner_link,
+                site,
+                q_id,
+                False,
+                False,
+                owner_rep,
+                post_score
+            )
             if is_spam:
                 try:
-                    handle_spam(title, body, owner_name, site, link, owner_link, q_id, reason, False, why, owner_rep, post_score, up_vote_count, down_vote_count, None)
+                    handle_spam(
+                        title,
+                        body,
+                        owner_name,
+                        site,
+                        link,
+                        owner_link,
+                        q_id,
+                        reason,
+                        False,
+                        why,
+                        owner_rep,
+                        post_score,
+                        up_vote_count,
+                        down_vote_count,
+                        None
+                    )
                 except:
-                    print "NOP"
+                    print "NOPE"
```

Comment:
The whole point of this was debugging in the beginning to note that nothing occurred here. (see https://en.wikipedia.org/wiki/NOP) Likely, we can just remove this, and make it a `pass` as @AWegnerGitHub suggests.